src/main/java/frc/robot/commandgroups/GoToPose.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.commandgroups;
+
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.controllers.PPHolonomicDriveController;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.HolonomicDriveController;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Robot;
+import frc.robot.RobotState;
+import swerve.SwerveDrive;
+
+public class GoToPose extends CommandBase {
```

Comment:
What is the purpose of this class? Does PathPlanner lack the functionality you need?